malaria-ios/AchievementProperty.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AchievementProperty.swift
+//  malaria-ios
+//
+//  Created by Teodor Ciuraru on 7/13/16.
+//  Copyright © 2016 Bruno Henriques. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+
+class AchievementProperty: NSManagedObject {
+
```

Comment:
Please provide a description on the current workflow. Anything to support and help the code review.